app/models/xccdf/benchmark.rb

```diff
@@ -29,6 +29,20 @@ def from_openscap_parser(op_benchmark)
       def policy_class
         BenchmarkPolicy
       end
+
+      def latest
+        select(
+          'DISTINCT ref_id, version, id, title'
```

Comment:
I had to do it like this because your suggestion would limit the fields retrieved to just ref_id. That's fine, but I need `version` later to compare versions en `Gem::Version` and also in GraphQL, a user should be able to query these fields at least (I didn't include description as we shouldn't need to query that for a SSG)